lambdas/criteria/evaluate.py

```diff
@@ -134,14 +158,62 @@ def get_candidate_evaluation_from_gpt(criteria, file_content, languages):
         ]
     )
     candidate_score = json.loads(completion.choices[0].message.content)
+    
     return candidate_score
```

Comment:
Since you have the explaination in the description of what you will return, I think you can just return directly
```
return json.loads(...)
```